internal/tools/protoc-gen-consul-rate-limit/main.go

```diff
@@ -5,29 +5,29 @@
 //
 // It works in two phases:
 //
-//	1. Buf/protoc invokes this plugin for each .proto file. We extract the rate
-//	   limit specification from an annotation on the RPC:
+//  1. Buf/protoc invokes this plugin for each .proto file. We extract the rate
+//     limit specification from an annotation on the RPC:
 //
-//	   service Foo {
-//	     rpc Bar(BarRequest) returns (BarResponse) {
-//	       option (hashicorp.consul.internal.ratelimit.spec) = {
-//	         operation_type: OPERATION_TYPE_WRITE,
-//	       };
-//	     }
-//	   }
+//     service Foo {
+//     rpc Bar(BarRequest) returns (BarResponse) {
+//     option (hashicorp.consul.internal.ratelimit.spec) = {
+//     operation_type: OPERATION_TYPE_WRITE,
+//     };
+//     }
+//     }
```

Comment:
Was this supposed to be included?

Reply:
it's the gofmt output, I will revert it.